src/In.ProjectEKA.OtpService/Otp/Logger/LogTemplate.cs

```diff
@@ -0,0 +1,7 @@
+namespace In.ProjectEKA.OtpService.Otp.Logger
+{
+    public class LogTemplate
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4954915)